services/src/test/scala/cromwell/pipeline/service/GitLabProjectVersioningTest.scala

```diff
@@ -1,52 +1,63 @@
 package cromwell.pipeline.service
+import akka.http.scaladsl.model.StatusCodes
 import cromwell.pipeline.datastorage.dao.repository.utils.TestProjectUtils
 import cromwell.pipeline.datastorage.dto.Project
+import cromwell.pipeline.utils.{ ApplicationConfig, GitLabConfig }
 import org.mockito.Mockito.when
-import org.scalatest.{ BeforeAndAfterAll, Matchers, WordSpec }
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, Matchers }
 import org.scalatestplus.mockito.MockitoSugar
 
-import scala.concurrent.{ ExecutionContext, Future }
+import scala.concurrent.Future
 
-class GitLabProjectVersioningTest extends WordSpec with Matchers with MockitoSugar with BeforeAndAfterAll {
-  implicit val executionContext: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
+class GitLabProjectVersioningTest
+    extends AsyncWordSpec
+    with ScalaFutures
+    with Matchers
+    with MockitoSugar
+    with BeforeAndAfterAll {
   val mockHttpClient: HttpClient = mock[HttpClient]
-  val gitLabProjectVersioning: GitLabProjectVersioning = new GitLabProjectVersioning(mockHttpClient)
+  val gitLabConfig: GitLabConfig = ApplicationConfig.load().gitLabConfig
+  val gitLabProjectVersioning: GitLabProjectVersioning = new GitLabProjectVersioning(mockHttpClient, gitLabConfig)
   import ProjectContext._
-  import GitLabConfig._
   "GitLabProjectVersioning" when {
     "createRepository" should {
+      val quote = "\""
+      def payload(project: Project): String =
+        s"{${quote}name${quote}:${quote}${project.ownerId.value}${quote},${quote}path${quote}:${quote}${project.projectId.value}${quote},${quote}visibility${quote}:${quote}private${quote}}"
```

Comment:
You can use single quote to not shield double quotes

Reply:
No, in JSON it must be exactly double-quoted.

Reply:
Use Json.toJson then

Reply:
Yes, I can, but this is test. So cause I use Json.toJson in the class realization in the test I use a different way.

Reply:
But this way it looks like you are testing Json.toJson method, but not your class)

Reply:
Yes, you right. I`ll fix it.

Reply:
Wow, what the hell is going on here? :)
Use triple quotes and that's it.
`s"""{"name":"${project.ownerId.value}","path":"${project.projectId.value}","visibility":"private"}"""`

Reply:
Thank you! Fixed.